API/Entities/BankAccount.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+namespace API.Entities
+{
+    public class BankAccount
+    {
+        public int Id { get; set; }
+
+        public int AppUserId { get; set; }
+
+        [Required]
+        [MaxLength(50)]
+        public string Name { get; set; }
+
+        public decimal Money { get; set; }
```

Comment:
A to pole od czego bedzie sluzyc? Od startowej wartosci konta? To wtedy zla nazwa. Czy bedzie trzymac aktualny stan konta? Czyli co kazda operacje bedziesz to z palca aktualizowal? Czy moze bedziesz stan konta wyliczal na biezaco na podstawie operacji